doc/sly.texi

```diff
@@ -553,7 +553,7 @@ Well, a good point to start is always the @emph{apropos} functionality,
 which is a @code{grep} of sorts, but symbolic rather than purely
 textual.  So if we want to hack on @SLY{}'s ``flex-completion'', so we
 type @kbd{C-c C-d C-z} (the shortcut for @kbd{M-x sly-apropos-all}) and
-then type in ``slyflex'' at the prompt, followed by @kbd{enter}
```

Comment:
When I wrote this, I probably really did intend to write "slyflex", since apropos at the time supported flex-matching itself. Are you telling me it's not working?

Reply:
Yes, I can confirm that `slyflex` does not work, but `sly.*flex` does. By the way, I’m using sly-20200228.1350.